lib/chef/formatters/error_inspectors/api_error_formatting.rb

```diff
@@ -112,10 +112,10 @@ def describe_406_error(error_description, response)
           max_server_version = version_header["max_version"]
 
           error_description.section("Incompatible server API version:", <<~E)
-            This version of the API that this Chef request specified is not supported by the Chef server you sent this request to.
+            This version of the API that this #{Chef::Dist::PRODUCT} client request specified is not supported by the Chef server you sent this request to.
```

Comment:
@mjingle @kagarmoe The existing wording here is funky right?

Reply:
What about `The version of the API that the request specified is not supported by the server`?

Reply:
@tas50 Definitely funky wording. For @marcparadise's suggestion, it looks good, but I would replace both instances of "the" with "this."

Reply:
I've taken a slighly different path, hopes this sounds ok for everyone :)